hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoTransferHandler.java

```diff
@@ -495,19 +504,42 @@ public Fees calculateFees(@NonNull final FeeContext feeContext) {
 
         /* Get subType based on the above information */
         final var subType = getSubType(
-                numNftOwnershipChanges, totalTokenTransfers, customFeeHbarTransfers, customFeeTokenTransfers);
-        return feeContext
+                numNftOwnershipChanges,
+                totalTokenTransfers,
+                customFeeHbarTransfers,
+                customFeeTokenTransfers,
+                triedAndFailedToUseCustomFees);
+        System.out.println("subType: " + subType);
+        final var ans = feeContext
                 .feeCalculator(subType)
                 .addBytesPerTransaction(bpt)
                 .addRamByteSeconds(rbs * USAGE_PROPERTIES.legacyReceiptStorageSecs())
                 .calculate();
+        System.out.println("  -> " + ans.totalFee());
+        return ans;
     }
 
+    /**
+     * Get the subType based on the number of NFT ownership changes, number of fungible token transfers,
+     * number of custom fee hbar transfers, number of custom fee token transfers and whether the transaction
```

Comment:
`and whether the transaction ` ...? Is this supposed to be the end of the sentence?

Reply:
fixed. thanks